src/schema/v2/searchCriteriaLabel.ts

```diff
@@ -435,3 +446,36 @@ async function getPartnerLabels(partnerIDs: string[], partnerLoader) {
     })
   )
 }
+
+async function getArtistSeriesLabels(
+  artistSeriesIDs: string[],
+  gravityGraphQLLoader
+) {
+  if (!gravityGraphQLLoader || !artistSeriesIDs?.length) return []
+
+  return Promise.all(
+    artistSeriesIDs.map(async (id) => {
+      const data = await gravityGraphQLLoader({
+        query: gql`
+          query GetArtistSeriesLabelsQuery($id: ID!) {
```

Comment:
This `gravityGraphQLLoader` was [the answer to the question](https://artsy.slack.com/archives/CP9P4KR35/p1699635639745649) of how to resolve upstream data from GravQL

Reply:
interesting!